inference-engine/src/readers/ir_reader/ie_ir_parser.cpp

```diff
@@ -453,7 +447,7 @@ std::shared_ptr<ngraph::Node> V10Parser::createNode(const std::vector<ngraph::Ou
         std::make_shared<LayerCreator<ngraph::op::Tanh>>("TanH"),
         std::make_shared<LayerCreator<ngraph::op::v0::Tile>>("Tile"),
         std::make_shared<LayerCreator<ngraph::op::TensorIterator>>("TensorIterator"),
-        std::make_shared<LayerCreator<ngraph::opset5::Loop>>("Loop"),
+        //std::make_shared<LayerCreator<ngraph::opset5::Loop>>("Loop"),
```

Comment:
Why did you comment Loop operator?

Reply:
My mistake, this file shouldn't be changed as no comment was addressed. Change reverted.